examples/lock-app/efr32/include/LockManager.h

```diff
@@ -158,6 +158,7 @@ class LockManager
                                 uint32_t localEndTime, DlOperatingMode operatingMode);
 
     bool IsValidUserIndex(uint16_t userIndex);
+    bool IsCredentialIndexGreaterThanZero(uint16_t credentialIndex, DlCredentialType type);
```

Comment:
His this prototype still needed then? Seems like you removed it in the source

Reply:
removed the unused function delclaration